### PR TITLE
Enable iOS and Android IAP SDK version override with Android X fix

### PR DESCRIPTION
**Pull request by hukid**
1. Enable iOS and Android SDK version override
1. Add docs for versioning
1. Fix AndroidX release build issue.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 ## Changelog
 
+### v1.1.1 Mar 27, 2019
+
+* Bump Square In-App Payments iOS SDK dependency to `1.1.1`.
+* Enable iOS and android In-App Payments SDK version override.
+* Fix #35 - support AndroidX release build
```
Comment (JohnMAustin78) [inline CHANGELOG.md]: We should link to [AndroidX](https://developer.android.com/jetpack/androidx/)

File: README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ In addition to this README, the following is available in the [flutter plugin Gi
 * [`doc`] - Root directory for all documentation.
 * [`example`] - Root directory of the Flutter sample app (with walkthrough).
 * [Getting started with the example app]
+* [Override the version of iOS and Android In-App Payments SDK]
```
Comment (JohnMAustin78) [inline README.md]: The title of the linked doc is "Versioning Strategy".  This list item should match the title of the linked doc.

File: doc/versioning.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Versioning strategy
```
Comment (JohnMAustin78) [inline doc/versioning.md]: Maybe make the title "Override the Default In-App Payments SDK Dependency Version"

File: doc/versioning.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+1. Remove `ios/Podfile.lock` and build your project again
+    ```bash
+    rm ios/Podfile.lock
+    flutter run
+    ```
+
+## Android
+
+1. Open `android/build.gradle` file, add the `sqipVersion` variable and specify your desired version
```
Comment (ellapolo) [inline doc/versioning.md]: Change to "Open the `android/build.gradle` file

File: doc/versioning.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        }
+    }
+
+    // add the override here
+    ext {
+        sqipVersion = '1.1.0'
+    }
+    ```
+
+1. clean the build and build your project again
```
Comment (ellapolo) [inline doc/versioning.md]: Capitalize the first word here.

File: doc/versioning.md (hunk 4 above)
Comment (ellapolo) [reply doc/versioning.md]: Also probably should add periods on the ends of all the sentences.

**Pull request merged**